### DISCUSSION TITLE
Shows too many blogs as not online

### DISCUSSION THREAD

**Discussion by johnflint1985** [TumblThreeApp/TumblThree#545] [General]
TumblThree sees too many blogs as not online, even though they are working. If I try to check if they are online it gives me this. 
"Unable to connect to remote server"

![issues 346366](https://github.com/TumblThreeApp/TumblThree/assets/29380159/9007731f-c81d-4c06-b5ef-4ba4d681d9a2)

**Comment by thomas694** [comment]
You had temporary connection problems or something like that. The blog and the 'online check' are working for now.
If the problem persists, please tell us again. Someone else can report it here too.
